docs/UserGuide.md

```diff
@@ -817,9 +817,9 @@ _________________________________________________________
 
 # <a id="db"></a> &nbsp;&nbsp; DB: `db`
 
-😃 What if you have forgotten all the database years that you have been using? Fear not! `db` command 
-is here to help! `db` command basically list out all the available database text file you currently
-have. 
+> 😃 What if you have forgotten all the database years that you have been using? Fear not! `db` command 
+> is here to help! `db` command basically list out all the available database text file you currently
```

Comment:
"basically list out" can change to "lists out"

Reply:
Ok!